docs/res/runner-auth-diags.txt

```diff
@@ -0,0 +1,49 @@
+# Markup used to generate the runner auth diagrams: https://websequencediagrams.com
+
+title Runner Configuration (self-hosted only)
+
```

Comment:
try this:
```
note left of Runner: GitHub repo URL as input
Runner->Github.com: Retrive Actions Service access using runner registration token
Github.com->Runner: Access token for Actions Service
note left of Runner: Generate RSA key pair
note left of Runner: Store encrypted RSA private key on disk
Runner->Actions Service: Register runner using Actions Service access token
note right of Runner: Runner name, RSA public key sent
note right of Actions Service: Public key stored
Actions Service->Token Service: Register runner as an OAuth application along with the RSA Public Key
note right of Token Service: Public key stored
Token Service->Actions Service: Client Id for the runner application
Actions Service->Runner: Client Id and Token Endpoint URL
note left of Runner: Store runner configuration info into .runner file
note left of Runner: Store Token registration info into .credentials file
```

Reply:
👍 I had to shorten out a few parts to get the diagram to fit, but I implemented most of the suggestions.